src/Bicep.Core.Samples/Files/InvalidExpressions_LF/Completions/azFunctions.json

```diff
@@ -44,23 +44,6 @@
       "command": "editor.action.triggerParameterHints"
     }
   },
-  {
-    "label": "listKeys",
```

Comment:
>listKeys [](start = 14, length = 8)

Why did this stop showing up in completions?

Reply:
See my previous comment about the method resolution cache. With a cache in place, listKeys gets added to the cache when it is first looked up in the document, and is then offered as a completion once it's in the cache.